recipes/unicorn-binance-suite/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "unicorn-binance-suite" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unicorn-binance-suite-{{ version }}.tar.gz
+  sha256: 04bad71ac74f92e454f0b8800289172b592a2dea5127ef2d19330c598da6c555
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - unicorn-binance-local-depth-cache
+    - unicorn-binance-rest-api
+    - unicorn-binance-websocket-api
+    - unicorn-fy
+
+test:
+  imports:
+    - unicorn_fy
+    - unicorn_binance_local_depth_cache
+    - unicorn_binance_rest_api
+    - unicorn_binance_websocket_api
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.lucit.tech/unicorn-binance-suite.html
+  summary: The UNICORN Binance Suite is a collection of open source Python packages that are useful for creating automated trading systems (bots) that connect to the Binance API.
```

Can you shorten this, maybe:
```suggestion
  summary: Suite of packages for creating automated trading systems with the Binance API.
```